yogstation/code/game/objects/items/premadepapers.dm

```diff
@@ -0,0 +1,53 @@
+//yogs code begins. Paperwork pre-made edition. What a pain this was. Most of these were taken from the wiki.
+/obj/item/paper/redd
+	name = "ERROR"
+	info = "You shouldn't have this."
+
+/obj/item/paper/redd/general_request_form
+	name = "General Requests Form (Form NT-010)"
+	fields = 3
+	info = "<b>Full Name:</b> <span class=\"paper_field\"></span><br><b>Request:</b> <span class=\"paper_field\"></span><br><b>Reason for Request: </b><span class=\"paper_field\"></span><br><br><u>Sign Below:<u>"
+
+//atmospheric technician spacesuit request form
+/obj/item/paper/redd/atmos_tech_spacesuit_form
+	name = "Atmos Spacesuit Request Form (Form NT-A14)"
+	fields = 1
+	info = "I, <b><span class=\"paper_field\"></span></b>, request a <b>space suit</b> from <b>EVA</b> in the event of an atmospheric emergency requiring one.<br><b>Signed,<br>Atmospheric Technician<br></b>"
```

Comment:
Make it a general item request form instead

Reply:
Also signature

Reply:
This is fine but there should also be a general item request one